.github/workflows/ci.yml

```
@@ -30,7 +30,9 @@ jobs:
           -DCLIENT_CERT_PATH="cert/path" \
           -DROOT_CA_CERT_PATH="cert/path" \
           -DCLIENT_PRIVATE_KEY_PATH="key/path" \
-          -DCLIENT_IDENTIFIER="ci-identifier"
+          -DCLIENT_IDENTIFIER="ci-identifier" \
+          -DS3_PRESIGNED_GET_URL="" \
+          -DS3_PRESIGNED_PUT_URL=""
```

Maybe we can try dummy URLs
```suggestion
          -DS3_PRESIGNED_GET_URL="s3.us-west-2.amazonaws.com" \
          -DS3_PRESIGNED_PUT_URL="s3.us-west-2.amazonaws.com"
```

---

What you put last works too